src/zone/napa-zone.h

```diff
@@ -42,6 +42,10 @@ namespace zone {
         /// <remark> Asynchronous works keep the reference on scheduler, so they can finish up safely. </remarks>
         std::shared_ptr<zone::Scheduler> GetScheduler();
 
+        // To avoid heavy dependecies on source files when build napa-binding on windows.
+        // After better build structure is used, remove this and change calls to the above.
+        NAPA_API static Scheduler* GetZoneScheduler(NapaZone* zone);
+
```

Comment:
Don't understand why this resolves build issue. looks like it's unnecessary.

Reply:
It is just related with windows build, keeping adding source files and keep report error. 
I think we need better build structure, and let it as different task. Not tracking here.